### DISCUSSION TITLE
Github Education Student ID

### DISCUSSION THREAD

**Discussion by spyke1215** [community/community#21118] [GitHub Education] [answered]
Hi I'm spyke, I submitted my ID, but the problem was, the validation date was on the back of my ID, so it don't have my name or face on it. I tried to take a photo on the front but i won't get submitted, then i tried to take a photo on the back and it submitted. I know that it will probably get rejected, so please i need help

**Comment by Ritocheta** [comment]
Hey spyke, I had the same issue with my ID card. What I did was I took photo of both the front and back of my ID card and made a pdf of them side by side and then took ss of them both together. After which I submitted this combined image using the upload photo option. If yours gets rejected you can try this out too. Hope this helps!

**Reply by spyke1215** [reply]
Hi @Ritocheta, How can i use the upload photo option. It only shows me to use my computer's camera
![image](https://user-images.githubusercontent.com/67054932/180644894-a404acd9-1ced-470b-aba2-c35ec5c78790.png)

**Reply by Ritocheta** [reply]
Hey spyke, you need to click on the take photo button first. Once you do that go to the bottom of your real time image and there on the left will be a button to click the photo and to the right will be the upload button which allows you to upload document from the laptop. At least that's how it worked for me. Hope this helps! Feel free to contact me if you still have doubt

**Reply by IamGroot05** [reply]
it not working there is no option for upload

**Comment by thuongtruong1009** [comment]
Hi @spyke1215 
The solution is you can use a scanner app to scan your photo to pdf and upload it. This will prevent the blur effect caused by light.

**Comment by kevinfinalboss** [comment]
I also had a problem with the student card

**Comment by c9long** [comment]
Hey Spyke, was wondering if you had an update on this/if anything in particular worked? The system's not accepting any of my IDs either, citing reasons such as the following when I submitted a photograph taken with my iphone camera:
![image](https://user-images.githubusercontent.com/109810526/181135220-6823b32c-f89e-478d-bb09-aeee0d5c373f.png)

**Comment by IamGroot05** [comment]
hi spyke i have a 100% working Trick . using this you can get approved . So 
    you just have to take your friend collage id (Who is studying with you in same collage ) just combine both id one below another in one time .it mean your id Front and your Friend id back side . now take photo both id at a time & submit. If it work give sime:)

**Comment by spyke1215** [comment, marked as answer]
The ID's didn't work for me, I just had another document for my school and that's what i submitted and it worked

**Comment by XYB3RSKY** [comment]
I didn't submit my student ID since they have not yet been processed but just submitted a screenshot of school's Exams website showing me logged into the website.

**Comment by Rasmondist** [comment]
No worries, we've all been there before. Don't worry about the validation date being on the back of your ID, there are ways to get around it!
One option is to use OCR technology, such as [Smart Engines OCR](https://smartengines.com/), to extract the text from the back of your ID and convert it into a format that can be submitted. This way, you can keep the information on the back of your ID but still be able to use it for your GitHub Education application.
Another option is to reach out to GitHub Education support and explain your situation. They may be able to provide you with alternative ways to verify your identity or assist you in submitting the ID in a way that works for both parties.
